variable/technology/power-plant.yaml

```
@@ -0,0 +1,799 @@
+# List of variables related to power plants
+# Variables related to the CO2 emission by power plant
+CO2 Emmissions|Electricity|Biomass|Traditionnal:
```

Typo in `Emissions` and `Traditional` made throughout this document

```suggestion
CO2 Emissions|Electricity|Biomass|Traditional:
```